src/utils/word/index.js

```diff
@@ -0,0 +1,23 @@
+export function selectWords(num, length) {
+    const data = require('../../data/words/' + length + '_letters_normal.json');
+    const ret = [];
+    const obj_keys = Object.keys(data);
+    const used_keys = [];
+
+    while(ret.length < num) {
+        var ran_key = obj_keys[Math.floor(Math.random() *obj_keys.length)];
+        if (!used_keys.includes(ran_key)){
+            used_keys.push(ran_key);
+            ret.push(data[ran_key]);
+        }
+    }
+
+    console.log(ret);
+    return ret;
+}
+
+export function findWord(word) {
+    const data = require('../../data/words/' + word.length + '_letters.json');
```

Comment:
So in node, you have the benefit of `require()` caching the results of the things it looks up.  I'm not sure how that translates to what webpack is gonna compile here.  Relating to my earlier comment about `require` in-line, it's possible that these are going to be included in the compiled app payload anyways.  Might be worth figuring out exactly what's happening at build time / run time.

Reply:
I'm just trying to access the data in that file is that doable without a `require`? I'm not sure what the suggestion is here

Reply:
`require()` doesn't exist in the browser, it's a `node` thing.  When webpack builds you app, it's going to _possibly_ inline these chunks.  I'm not 100% sure. What I simply mean is, "What's gonna happen when webpack compiles this and can we just `require` everything at the top of the file?"